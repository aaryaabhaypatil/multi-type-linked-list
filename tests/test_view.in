NEW 3
comp2017
malloc
realloc
VIEW 0
NEW 1
2
REMOVE 0
VIEW 1
VIEW all
VIEW 0