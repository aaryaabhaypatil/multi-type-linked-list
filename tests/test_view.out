List 0: comp2017 -> malloc -> realloc
comp2017 -> malloc -> realloc
List 1: 2
List 0 has been removed.

Number of lists: 1
List 1
2
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW
